cesium_app/handlers/model.py

```diff
@@ -107,6 +110,8 @@ def post(self):
             params_to_optimize=params_to_optimize)
         score_future = executor.submit(build_model.score_model, computed_model,
                                        imputed_fset)
+        best_params = executor.submit(lambda cmp_mod: cmp_mod.best_params_,
+                                      computed_model) if params_to_optimize else {}
```

Comment:
Maybe remove the `impute` call and just assume that the featureset was properly imputed before?